src/main/resources/resource/BoofCV/visualdepth.yaml

```diff
@@ -0,0 +1,20 @@
+# RGB Depth Camera Calibration
```

Comment:
why is this file checked in?  what uses it?  I see no references to it in the code base.